Design/UseCases.md

```diff
@@ -0,0 +1,80 @@
+## Actors
+* **Software System** - Payment management, Keyword analysis, Pitch cataloging, Collect statistical data.
+
+* **Founder** - Pitch creation, Create statistics.
+
+* **Investor** -Observation of parties, provide funds.
+
+* **Developer** - Completion of tasks, observation of parties.
+
+* **User (Founder, Investor, and Developer)** - Profile creation, Sending direct messages, Negotiate terms, Fluid role choice.
+
+## Use Cases
+* **UC1** Search for projects.
+  * Searching for projects is a fundamental and necessary part of the system. It will allow the actors of our system to find different projects to inquire about. They may not join them, but having the ability to search for them will give them the opportunity to find projects they prefer.
+  * To use this function the actor will locate the search function on the page and filter for the project their looking for by name.
+  * Actors: Developer, Investor, System
+  * BR1
+  
+* **UC2** Define keywords for pitch.
+  * This is a use case because the founder will achieve the outcome of defining keywords for a pitch. Certain keywords are associated with different types of pitches. Being able to define by keywords for a pitch during pitch creation is important because it gives it a category for interested users to search in. Founders will search for pitches they're interested in based on those keywords. 
+  * Inside of the project creation menu there will be a text box titled "Keywords". The founder can select this text box to enter 
+  * Actors: Founder
+  * BR1
+
+* **UC3** Create user profiles.
+  * This is a use case because the user will achieve the outcome of creating a user profile. The users will need profiles to store different information related to their project and the site. They can also use them to communicate, see graphs, and create or join projects.
+  * In order to create a user profile they will have to navigate to the "Sign Up" button, click it, and then fill out a form asking for their email and a password. After they fill out the form and hit "Submit", they'll have created a user profile.
+  * Actors: User
+  * BR2
+  
+* **UC4** Create statistical graphics.
+  * This is a use case because the user will achieve the outcome of creating graphs. Statistical graphs will be used to represent data about a project. It will provide useful information that can be used to see if the project is on a good path or not. It will beneficial in the sense that it provides
+  * The system will create the graphs under the Graphs page. It will collect the input over the course of the project.
+  * Actors: System, Founder
+  * BR2
+  
+* **UC5** Format pitch page.
```

Comment:
For this one I would specify that typically you are only formatting a pitch page under the user role of founder. Then maybe explain in depth the specifics of the founder role